cvat/apps/engine/media_extractors.py

```diff
@@ -266,7 +268,8 @@ def __init__(self,
 
         self._archive_source = source_path[0]
         tmp_dir = extract_dir if extract_dir else os.path.dirname(source_path[0])
-        Archive(self._archive_source).extractall(tmp_dir)
+        patool_path = os.path.join(sys.exec_prefix, 'bin', 'patool')
```

Comment:
@azhavoro Could you suggest maybe there is a better way to acess `.env/bin/patool` executable?

Reply:
Why we need here to pass `patool_path` arg? Anyway, I'll check

Reply:
Consider using `sysconfig.get_path('scripts')`.